app/lib/submission_builder/ty2022/states/ny/documents/it201.rb

```diff
@@ -94,7 +95,7 @@ def document
                   intake.dependents.each do |dependent|
                     xml.depInfo do
                       xml.DEP_CHLD_FRST_NAME claimed: dependent.first_name
-                      xml.DEP_CHLD_MI_NAME claimed: dependent.middle_initial
+                      xml.DEP_CHLD_MI_NAME claimed: dependent.middle_initial if dependent.middle_initial
```

Comment:
I think .present? here would prevent empty strings from causing this to show

Reply:
Good call, updated